include/gepetto/gui/mainwindow.hh

```diff
@@ -155,7 +156,7 @@ signals:
 
         void connectSlot(const char *slot, const char *signal, QObject* obj);
 
-        /// \}
+      void registerShortcut(QString widgetName, QString actionName, QAction* action);
```

Comment:
See line note in https://github.com/jmirabel/hpp-plot/pull/5#discussion_r71306052